### PR TITLE
NO-TICKET: various updates

### PR DESCRIPTION
**Pull request by Fraser999**
This updates our dependencies, in particular those flagged as insecure at https://deps.rs/repo/github/CasperLabs/casper-node.

It also brings the three chainspec files in closer alignment with each other.

### CODE REVIEW

File: resources/charlie/chainspec.toml [hunk 1]
```diff
@@ -111,38 +111,38 @@ gas_per_byte = 630_000_000
 
 # Host function declarations are located in smart_contracts/contract/src/ext_ffi.rs
 [wasm_config.host_function_costs]
-add = { cost = 5_800_000, arguments = [0, 0, 0, 0] }
+add = { cost = 5_800_000 ,arguments = [0, 0, 0, 0] }
```
Comment (EdHastingsCasperLabs) [inline resources/charlie/chainspec.toml:114]: was this one intentional?

File: resources/local/chainspec.toml [hunk 2]
```diff
@@ -69,87 +69,87 @@ initial_memory = 64
...
-add_contract_version = { cost = 200_000_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9] } # Not instrumented yet, assuming a sufficiently large number
-blake2b = { cost = 200_000, arguments = [0, 1, 2, 3] }
-call_contract = { cost = 4_500_000, arguments = [0, 1, 2, 3, 4, 5, 6] }
-call_versioned_contract = { cost = 4_500_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7, 8] } # Not instrumented yet, assuming the same as call_contract
-create_contract_package_at_hash = { cost = 200_000_000, arguments = [0, 1] } # Not instrumented yet, assuming a sufficiently large number
-create_contract_user_group = { cost = 200_000_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7] } # Not instrumented yet, assuming a sufficiently large number
-create_purse = { cost = 170_000_000, arguments = [0, 1] }
-disable_contract_version = { cost = 200_000_000, arguments = [0, 1, 2, 3] } # Not instrumented yet, assuming a sufficiently large number
-get_balance = { cost = 3_800_000, arguments = [0, 1, 2] }
+add = { cost = 5_800_000 ,arguments = [0, 0, 0, 0] }
```
Comment (EdHastingsCasperLabs) [inline resources/local/chainspec.toml:114]: ?

File: utils/nctl/templates/chainspec.toml [hunk 3]
```diff
@@ -58,99 +58,99 @@ max_ttl = '1day'
...
-add_contract_version = { cost = 200_000_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9] } # Not instrumented yet, assuming a sufficiently large number
-add_local = { cost = 4_700_000, arguments = [0, 1, 2, 3] }
-call_contract = { cost = 4_500_000, arguments = [0, 1, 2, 3, 4, 5, 6] }
-call_versioned_contract = { cost = 4_500_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7, 8] } # Not instrumented yet, assuming the same as call_contract
-create_contract_package_at_hash = { cost = 200_000_000, arguments = [0, 1] } # Not instrumented yet, assuming a sufficiently large number
-create_contract_user_group = { cost = 200_000_000, arguments = [0, 1, 2, 3, 4, 5, 6, 7] } # Not instrumented yet, assuming a sufficiently large number
-create_purse = { cost = 170_000_000, arguments = [0, 1] }
-disable_contract_version = { cost = 200_000_000, arguments = [0, 1, 2, 3] } # Not instrumented yet, assuming a sufficiently large number
-get_balance = { cost = 3_800_000, arguments = [0, 1, 2] }
+add = { cost = 5_800_000 ,arguments = [0, 0, 0, 0] }
```
Comment (EdHastingsCasperLabs) [inline utils/nctl/templates/chainspec.toml:115]: ...

**Review by EdHastingsCasperLabs — APPROVED** [review]
A very minor nit, otherwise lgtm

**Review by momipsl — APPROVED** [review]

**Pull request merged**